pilots/10_things_to_know_about_pilots.Rmd

```diff
@@ -0,0 +1,189 @@
+  Pilot = c("10% reported difficulty.", "75% correctly identified the politician party.", "Participants were more familiar with some politicians in the videos than others.", "Respondents appear to have been confused by bidirectional scales and questions with reversed scales."),
+  Design = c("We will add subtitles to the video treatments.", "We will mention the politician party additional times to make this treatment component more memorable to participants.", "We will randomize subjects to a more familiar or less familiar politician and may explore heterogeneous effects by politician familiarity.", "We will use unidirectional scales and also consider the distribution (variance) of other key outcome measures.")
+  )
+names(tab3) <- c("Question", "Pilot Results", "Design Decision")
+
+panderOptions("table.continues", "")
+pander::pander(tab3, keep.line.breaks = T, style = "grid", justify = "left", split.table = c("20%", "20%", "20%"))
+```
+
+# 9. Explore unknowns
+
+While you may begin your study with a set of preestablished questions---or "known unknowns"--- that your pilot can help to address, keep in mind that there is a whole universe of "unknown unknowns" left to be explored. Of course, not all of these will be relevant to your study, but some are likely to be. For example, you may find that the participants have vastly different interpretations of an informational treatment, understand survey scales differently, or may be reluctant to share responses given a perceived political bias.
+
+There are a few ways to explore these unknowns in your pilot study. You might wish to include additional exploratory outcome questions, collect extra covariate data, or use a variety of treatments drawing on innovative or untested ideas. Another great way to identify unknowns is through open-ended questions of study participants. Regardless of your study design, you might consider surveying or interviewing participants to ask "what they think about topic X," or "what comes to mind when they hear the term Y." The world is often more complex than researchers model in their study designs, and study participants often have more diverse perspectives and relationships with the issues at hand than researchers expect. 
+
+Through this process, you may identify novel research questions, potential theoretical dimensions or causal mechanisms, and hypotheses that you had not originally formulated when you began studying the topic. Exploring unknowns can therefore lead to refining the ideas you had originally conceived of, as you are unlikely to have determined the best version of your study from the beginning. It can also lead to entirely novel considerations and ideas altogether, some of which you may be able to incorporate into the current study or future studies.
+
+Overall, pilots afford a wonderful opportunity for open-ended exploration and idea generation that should not be overlooked.
+
+# 10. A pilot is part of a broader sequence of research activities

```

Comment:
One idea is to move this #10 to the new #1 that I proposed --- so that people who are not sure what you are talking about with pilot studies can orient themselves. Also it helps you set scope: "We are talking about pilot studies mostly in the context of randomized field experiments...."